nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/elasticsearch/ElasticsearchObject.java

```diff
@@ -64,7 +65,51 @@ public void doIndexing(@PathParam("repositoryName") String repositoryName) {
         });
     }
 
+    /**
+     * Executes an ES flush on all documents indexes of a given repository.
```

Comment:
on the document index of a given repository (there's only one document index per repository)